remote/remote_state_gcs.go

```diff
@@ -142,7 +142,8 @@ func gcsConfigValuesEqual(config map[string]interface{}, existingBackend *Terraf
 	}
 
 	if !terraformStateConfigEqual(existingBackend.Config, comparisonConfig) {
-		terragruntOptions.Logger.Printf("Backend config has changed from %s to %s", existingBackend.Config, config)
+		terragruntOptions.Logger.Printf("Backend config has changed")
```

Comment:
Sorry, one more NIT, but shouldn't this statement also say "Set environment variable TF_LOG=debug to have..."?

Reply:
Gah sorry for sloppiness... It's been a while since I opened this PR and forgot there were more places. This is now fixed: a3d396c